src/main/webapp/availabilityTable.jsp

```diff
@@ -1,12 +1,15 @@
 <%@ page import="com.google.sps.data.AvailabilityTimeSlotGenerator" %>
 <%@ page import="com.google.sps.data.AvailabilityTimeSlot" %>
 <%@ page import="com.google.sps.data.DatastoreAvailabilityDao" %>
-<%@ page import="java.util.List,java.time.Instant" %>
+<%@ page import="java.util.List" %>
+<%@ page import="java.time.Instant" %>
+<%@ page import="java.time.temporal.ChronoUnit" %>
 <%@ page import="java.lang.Integer" %>
 <%
   List<List<AvailabilityTimeSlot>> list = AvailabilityTimeSlotGenerator
-    .timeSlotsForWeek(Instant.now(), Integer.parseInt(request
-    .getParameter("timeZoneOffset")), new DatastoreAvailabilityDao());
+      .timeSlotsForWeek(Instant.now().plus((Integer.parseInt(request
+      .getParameter("page")) * 7), ChronoUnit.DAYS), Integer.parseInt(request
```

Comment:
Fix indentation.

Reply:
Claire was asking for this to be indented twice because it's the rest of the original line. Should it only be indented once?

Reply:
These should not be aligned:
```java
      .timeSlotsForWeek(Instant.now().plus((Integer.parseInt(request
      .getParameter("page")) * 7), ChronoUnit.DAYS), Integer.parseInt(request
```

They're at different levels of nesting. The same applies below. The alignment here is far from helpful -- it's actually actively misleading.

Here's what this should probably look like:

```java
  List<List<AvailabilityTimeSlot>> list = 
    AvailabilityTimeSlotGenerator.timeSlotsForWeek(
        Instant.now().plus(
            7 * Integer.parseInt(request.getParameter("page")), 
            ChronoUnit.DAYS), 
        Integer.parseInt(request.getParameter("timeZoneOffset")), 
        new DatastoreAvailabilityDao());
```

The significant part is that the line breaks happen between parameters. All levels of nesting are consistent within the argument set that they are a part of.

That said, by the way, if any of those `parse ints` you'd be causing a headache for someone to debug. Generally I support inlining, but when the things being inlined can throw their own exceptions (and there are several of them), then you can cause a bit of code to become a hassle to debug.

Reply:
Handled.